systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationST.java

```diff
@@ -231,8 +228,6 @@ void testTeamAReadFromTopic(ExtensionContext extensionContext) {
     @Order(4)
     void testTeamBWriteToTopic(ExtensionContext extensionContext) {
         String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
-        String teamBProducerName = TEAM_B_PRODUCER_NAME + "-" + clusterName;
-        String teamBConsumerName = TEAM_B_CONSUMER_NAME + "-" + clusterName;
```

Comment:
what about changing it in the `KafkaOauthExampleClients.Builder()`?

Reply:
Done